cmake/FindLibc.cmake

```diff
@@ -7,10 +7,17 @@ if(LIBC_LIBRARIES)
     set(LIBC_FIND_QUIETLY TRUE)
 endif(LIBC_LIBRARIES)
 
-find_library(librt NAMES rt)
-find_library(libdl NAMES dl)
-find_library(libm NAMES m)
-set(LIBC_LIBRARIES ${librt} ${libdl} ${libm})
+# TODO OSX see if rt on Linux needed really
```

Comment:
rt contains some aio and timer symbols. Could you update the comment. And maybe put dl/m outside of the if/else